Automattic/wp-calypso
# Framework: Disable logged-out masterbar logo link

Fixes #5419

Follow-up from #5453.

Properly routing to the logged-out wordpress.com homepage would require a server-reload. Just disable for now.

**To Test**
* Go to http://calypso.localhost:3000/start or http://calypso.localhost:3000/design
* The WordPress.com logo on the top bar should not be a link
